internal/httpcontroller/handlers/settings.go

```diff
@@ -152,8 +160,8 @@ func (h *Handlers) updateAuthenticationSettings(settings *conf.Settings) {
 		}
 	}
 
-	if (settings.Security.GoogleAuth.Enabled || settings.Security.GithubAuth.Enabled || basicAuth.Enabled) &&
-		settings.Security.SessionSecret == "" {
+	// Generate a random session secret for Gothic
+	if settings.Security.SessionSecret == "" {
```

Comment:
Prevent unnecessary host address validation error on UI, when authentication is disabled.